packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -296,16 +285,12 @@ const ModalManager = () => {
         is_mt5_password_invalid_format_modal_visible ||
         is_sent_email_modal_enabled;
 
-    const is_invalid_investor_token =
-        Object.keys(current_list).length === 0 && localStorage.getItem('cfd_reset_password_code');
-    const should_show_cfd_reset_password_modal = is_cfd_reset_password_modal_enabled && !is_invalid_investor_token;
```

Comment:
You'd better keep the validation check of the `code` which we use in the user redirection